docs/root/intro/arch_overview/outlier.rst

```diff
@@ -70,12 +70,14 @@ with the minimum required request volume in an interval is less than the
 :ref:`outlier_detection.success_rate_minimum_hosts<envoy_api_field_cluster.OutlierDetection.success_rate_minimum_hosts>`
 value.
 
+.. _arch_overview_outlier_detection_logging:
```

Comment:
what's the story here?

Reply:
@jmarantz updated below section according to new proto schema